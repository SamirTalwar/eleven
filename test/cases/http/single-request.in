http --check-status --print='hb' :8080 | dos2unix | grep -Ev '^Date: '
