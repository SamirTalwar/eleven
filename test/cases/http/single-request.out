HTTP/1.1 200 OK
Connection: keep-alive
Transfer-Encoding: chunked

Hello, world!
